app-arch/torcx/torcx-9999.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) 2017 CoreOS, Inc.. All rights reserved.
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+CROS_WORKON_PROJECT="coreos/torcx"
+CROS_WORKON_LOCALNAME="torcx"
+CROS_WORKON_REPO="git://github.com"
+COREOS_GO_PACKAGE="github.com/coreos/torcx"
+COREOS_GO_VERSION="go1.8"
+
+if [[ "${PV}" == 9999 ]]; then
+	KEYWORDS="~amd64 ~arm64"
+else
+	CROS_WORKON_COMMIT="1987d1c516c6925280fade659569d5062999a1db" # v0.1.0-alpha.1a
+	KEYWORDS="amd64 arm64"
+fi
+
+inherit coreos-go cros-workon systemd
+
+DESCRIPTION="torcx is a boot-time addon manager for immutable systems"
+HOMEPAGE="https://github.com/coreos/torcx"
+LICENSE="Apache-2.0"
+SLOT="0"
+
+src_compile() {
+	CGO_ENABLED=0 go_export
+	${EGO} build -v \
+		-p "$(makeopts_jobs)" \
+		-ldflags "-X ${COREOS_GO_PACKAGE}/pkg/version.VERSION=${PV}" \
+		-o "bin/${ARCH}/torcx" \
+		-tags containers_image_openpgp \
+		"${COREOS_GO_PACKAGE}"
+}
+
+src_install() {
+	local generatordir=/usr/lib/systemd/system-generators
+	local vendordir=/usr/share/torcx
+
+#	dobin "${S}/bin/${ARCH}/torcx"
+
+	exeinto "${generatordir}"
+	newexe "${S}/bin/${ARCH}/torcx" torcx-generator
+#	mkdir -p "${ED}${generatordir}"
+#	ln -fns ../../../bin/torcx "${ED}${generatordir}/torcx-generator"
+	systemd_dounit "${FILESDIR}/torcx.target"
+
+	insinto "${vendordir}/profiles"
+	doins "${FILESDIR}/vendor.json"
+	mkdir -p "${ED}${vendordir}/store"
```

Comment:
It was to install it in the buildroot correctly.  I've just changed it to `dodir` instead.